app/src/main/java/com/zulip/android/networking/GravatarAsyncFetchTask.java

```diff
@@ -47,7 +48,7 @@ public static URL sizedURL(Context context, String url, float dpSize) {
         float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP,
                 dpSize, r.getDisplayMetrics());
         try {
-            return new URL(url + "&s=" + px);
+            return new URL(ZulipApp.get().getServerHostUri() + url + "&s=" + px);
```

Comment:
Just to confirm: Zulip sends both relative URLs (for custom avatars) and absolute URLs (for default gravatars); does this code handle both cases?